packages/bson-bench/tsconfig.json

```
@@ -0,0 +1,45 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
```

```suggestion
    "allowJs": false,
```

@baileympearson we never flipped this flag back in the driver after FLE 😅 anyway no need to introduce it here.